bundles/automation/org.eclipse.smarthome.automation.module.script.rulesupport/src/main/java/org/eclipse/smarthome/automation/module/script/loader/internal/file/ScriptFileProvider.java

```diff
@@ -0,0 +1,217 @@
+/**
```

Comment:
why can this file be removed? It is used to load scripts? The Thread can be replaced with a task for the scheduler, but I don't see where it can be completely removed?